src/common/json-schema-input/generic-input.js

```diff
@@ -56,7 +58,10 @@ const InputByType = {
   remote: XoRemoteInput,
   sr: XoSrInput,
   string: StringInput,
-  vm: XoVmInput
+  vm: XoVmInput,
+  xoobject: XoHighLevelObjectInput,
```

Comment:
I remember seeing a use of the XoSubject term somewhere, but I don't remember where right now, I'll recheck and change this if it doesn't pull up any question.